### ISSUE TITLE
Pm2-health send too many emails at once when a program died or exited.

### ISSUE THREAD

**Issue by just-pthai-it** [pankleks/pm2-health#51]
Just like the title. I have problem with number of emails sent. But I did not find any solution to fix it. Already adjust batchPeriodM: 5 and batchMaxMessages:20 but nothing change.

**Comment by ZhuBoao** [comment]
Yes it's an devil now, sending more than 100 emails per seconds and I have to turn it off.

**Comment by gullyboy007** [comment]
Are there any alternative to pm2-health ? @ZhuBoao @just-pthai-it  Or how to overcome this issue? I am planning to add pm2-health to my repo.

**Comment by just-pthai-it** [comment]
> Are there any alternative to pm2-health ? @ZhuBoao @just-pthai-it  Or how to overcome this issue? I am planning to add pm2-health to my repo. 

Sorry, I have not found any solution for this case
